test-suite/src/function/splice.rs

```diff
@@ -0,0 +1,60 @@
+use {
+    crate::*,
+    gluesql_core::{executor::EvaluateError, prelude::Value::*},
+};
+
+test_case!(splice, async move {
```

Comment:
now all the test-suite refactoring works which I was planning to do at this time are finished and merged into main branch.
there's some changes on writing tests.
1. `async move {` -> `{`
2. `run!`, `test!` macros -> `g.run(..`, `g.test(..` normal functions

could you merge & apply these changes?